test/parallel/test-options-binding.js

```diff
@@ -0,0 +1,21 @@
+// Flags: --expose-internals
+'use strict';
+
+const common = require('../common');
+const { primordials } = require('internal/test/binding');
+const {
+  SafeMap,
+} = primordials;
```

Comment:
nit: we could make it a bit more concise.
```suggestion
const { primordials: { SafeMap } } = require('internal/test/binding');
```